ArchiSteamFarm/ArchiWebHandler.cs

```diff
@@ -133,6 +133,41 @@ internal sealed class ArchiWebHandler : IDisposable {
 			return await UrlPostWithSession(SteamStoreURL, request, data).ConfigureAwait(false);
 		}
 
+		internal async Task<bool> ChangePrivacySettings(Steam.PrivacyResponse privacySettings) {
+			string request = GetAbsoluteProfileURL() + "/ajaxsetprivacy";
+
+			// We have to do this because Steam uses the same numbers for settings except comments, this has its special numbers, fuck Steam
+			string commentPermission = "";
```

Comment:
It'd be better to define `ECommentPermission` type somewhere in `Steam.PrivacyResponse` and use it instead. This parsing is good, but you should parse it to `ECommentPermission` and now to a `string`.